api/grpc/server.go

```diff
@@ -645,33 +645,33 @@ func (a *PayChServer) ClosePayCh(ctx context.Context, req *pb.ClosePayChReq) (*p
 	if err != nil {
 		return errResponse(err), nil
 	}
-	channel, err := sess.GetCh(req.ChannelID)
+	ch, err := sess.GetCh(req.ChID)
 	if err != nil {
 		return errResponse(err), nil
 	}
-	payChInfo, err := payment.ClosePayCh(ctx, channel)
+	payChInfo, err := payment.ClosePayCh(ctx, ch)
 	if err != nil {
 		return errResponse(err), nil
 	}
 
 	return &pb.ClosePayChResp{
 		Response: &pb.ClosePayChResp_MsgSuccess_{
 			MsgSuccess: &pb.ClosePayChResp_MsgSuccess{
-				ClosingBalance: ToGrpcBalInfo(payChInfo.BalInfo),
-				ClosingVersion: payChInfo.Version,
+				ClosedBalInfo: ToGrpcBalInfo(payChInfo.BalInfo),
+				ClosedVersion: payChInfo.Version,
 			},
 		},
 	}, nil
 }
 
 // FromGrpcBalInfo is a helper function to convert BalInfo struct defined in grpc package
 // to BalInfo struct defined in perun-node. It is exported for use in tests.
-func FromGrpcBalInfo(src *pb.BalanceInfo) perun.BalInfo {
+func FromGrpcBalInfo(src *pb.BalInfo) perun.BalInfo {
 	balInfo := perun.BalInfo{
 		Currency: src.Currency,
-		Bals:     make(map[string]string, len(src.Balances)),
+		Bals:     make(map[string]string, len(src.Bals)),
 	}
-	for _, aliasBalance := range src.Balances {
+	for _, aliasBalance := range src.Bals {
```

Comment:
Please change `aliasBalance` to `aliasBal`
```suggestion
	for _, aliasBal := range src.Bals {
```